generators/app/index.js

```diff
@@ -80,5 +88,9 @@ module.exports = yeoman.Base.extend({
       var testPath = folder + '/__test__/index.js';
       this.template('test.js', testPath, data);
     }
+    if (props.addActions) {
+      var actionsPath = folder + '/' + data.name + 'Actions.js'
```

Comment:
What if we just use `actions.js`  as it's already inferred based on the folder it sits inside of? I'm almost thinking we could do the same for the actual component and export `index.js` in the [package.json](https://github.com/visormatt/generator-react-up/blob/master/generators/app/templates/package.json#L5) vs. name & entry like it is now?

**Pros:** more concise and [standard](https://github.com/substack/node-browserify/blob/master/package.json#L5)
**Cons:** less obvious the component.js 

* We already have this info in the package.json _name_ & _folder_